core/src/main/tut/README.md

```diff
@@ -7,20 +7,53 @@
+PureConfig is a Scala library for loading configuration files. It reads [Typesafe Config](https://github.com/typesafehub/config) configurations written in [HOCON](https://github.com/typesafehub/config/blob/master/HOCON.md#hocon-human-optimized-config-object-notation), Java `.properties`, or JSON to native Scala classes in a boilerplate-free way. Sealed traits, case classes, collections, optional values, and many other [types are all supported out-of-the-box](https://pureconfig.github.io/docs/built-in-supported-types.html). Users also have many ways to add support for custom types or customize existing ones.
 
 Click on the demo gif below to see how PureConfig effortlessly translates your configuration files to well-typed objects without error-prone boilerplate.
 <br clear="right"> <!-- Turn off the wrapping for the logo image. -->
 
 ![](http://i.imgur.com/P6sda06.gif)
 
-## Documentation
 
-Read the [documentation](https://pureconfig.github.io/docs).
+## Why
+
+Loading configurations has always been a tedious and error-prone procedure. A common way to do it
+consists in writing code to deserialize each fields of the configuration. The more fields there are,
+the more code must be written (and tested and maintained...) and this must be replicated for each project.
+
+This kind of code is boilerplate because most of the times the code can be automatically generated by
+the compiler based on what must be loaded. For instance, if you are going to load an `Int` for a field
+named `foo`, then probably you want some code that gets the values associated with the key `foo` in
+the configuration and assigns it to the proper field after converting it to `Int`.
+
+The goal of this library is to create at compile-time the boilerplate necessary to load a configuration of a
+certain type. In other words, you define **what** to load and PureConfig provides **how** to load it.
+
+
+## Quick Start
 
-## Use PureConfig
+To use PureConfig in an existing SBT project with Scala 2.10 or a later version, add the following dependency to your
+`build.sbt`:
 
-First, import the library, define data types, and a case class to hold the configuration:
+```scala
+libraryDependencies += "com.github.pureconfig" %% "pureconfig" % "0.8.0"
+```
+
+For Scala 2.10 you need also the Macro Paradise plugin:
+
+```scala
+libraryDependencies ++= Seq(
+  "com.github.pureconfig" %% "pureconfig" % "0.8.0",
+  compilerPlugin("org.scalamacros" % "paradise" % "2.0.1" cross CrossVersion.patch)
+)
+```
+
+For a full example of `build.sbt` you can have a look at this [build.sbt](https://github.com/pureconfig/pureconfig/blob/master/example/build.sbt).
+
+Earlier versions of Scala had bugs which can cause subtle compile-time problems in PureConfig.
+As a result we recommend only using the latest Scala versions within the minor series.
+
+In your code, first import the library and define data types and a case class to hold the configuration:
 
 ```scala
 import pureconfig.loadConfig
```

Comment:
Nitpick: I suggest removing this line and making the call to `loadConfig` below fully qualified or placing the import closer to the usage.
Feel free to ignore.

Reply:
Yep, I agree. But if you don't mind, I'll do it in my next PR. As I mentioned on the PR description, I think it's better for reviewing if we set out the structure first and update/fix the content afterwards separately. Moving files and sections around like this always leads to very messy diffs (as you can see in the "Files changed" tab) and I usually avoid doing changes (even small ones) since they get easily overlooked by reviewers.